### DISCUSSION TITLE
PGMA on Windows was working Fine - Until I Updated PMS

### DISCUSSION THREAD

**Discussion by jfreiman** [CodyBerenson/PGMA-Modernized#236] [Q&A] [answered]
Everything on PMS was working as expected until I upgraded to the latest beta version of the software - then none of the agents were matching.
After working with JPH and none of his suggestions worked (and he confirmed his same version was working) at his suggestion I uninstalled PMS, deleted the files in the plug-ins folder, rebooted Windows and re-installed PMS and then PGMA.
That worked, until I updated to the next beta version, then PGMA stopped working again.
After a week, I'm finally getting around to uninstalling and reinstalling PMS (after uninstalling and rebooting), ran Plex and confirmed that PGMA was not listed in the Agents.
I closed PMS, copied the latest PGMA (~April 23, 2023) into the Plug-Ins folder and restarted PMS.
After several minutes all the agents in PGMA appeared and I went through the active agents, opened them and saved them.
The trouble is, my first priority agent is _Internet Adult Film Database_ and then attempt to match the films I've added in the past week, none of them are matching to IAFD.  They are matching with _Gay Hot Movies_ which ranked #2 in my Film Agents.
If I choose a film already matched by IAFD (from 2+ weeks ago) and Refresh the data, it/they still do not match IAFD and matches to the _GEVI_ which is ranked #5 (and last) agent.

I've tried manually moving the ranking and moving them back to my preferred order again to no avail.
[PMS Plugin Logs.zip](https://github.com/CodyBerenson/PGMA-Modernized/files/11358363/PMS.Plugin.Logs.zip)

I didn;t mention this above, but no matter which agent is used to match, the only Cast artwork being added is the "IAFD No Phot Available" artwork.

I'd really appreciate it if someone could take a look at my logs and tell me what is broken and how to resolve this issue.

Thanks in advance,

John

**Comment by jfreiman** [comment]
here is an example of a title that is not matching 
(Pantheon Productions) - Cruising Daddies (2023).mp4
https://www.iafd.com/title.rme/title=cruising+daddies/year=2023/cruising_daddies.htm

and I checked to verify that the Plex Token was correct and not blank.  I also checked the Plex Token against the one entered in the _Gay Hot Movies_ token and they match.

**Comment by CodyBerenson** [comment, marked as answer]
see discussion topic #232.

**Reply by jfreiman** [reply]
😭
